core/src/main/java/hudson/cli/CLICommand.java

```diff
@@ -642,4 +653,10 @@ public static CLICommand getCurrent() {
             }
         }
     }
+
+    public static class CLIUserDetails extends User {
```

Comment:
Needs Javadoc.

Reply:
JavaDoc added but also restricted on the class since it's only used by CLI and is not expected to be used by someone else.